### PR TITLE
9.18 RN

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/en/docs/releasenotes/studio-pro/9/9.18.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+We see the biggest value here for Design mode, as it gives this mode more control. However, it can be used in combination with Structure mode as well.
+
+With a default Studio Pro layout, the new pane can be found on a tab next to the **App Explorer**. Alternatively, it can be activated in the **View** menu.
+
+#### Widget Developer Console
+
+We added a new feature called the Widget Developer Console for advanced widget development, which helps you debug the configuration code of your widget that is responsible for conditionally visible properties, custom consistency checks, and Structure mode preview. This console shows logs from the `getProperties`, `getPreview`, `check`, and `getCustomCaption` functions, which can be defined inside the new configuration module (for more information, see [](/apidocs-mxsdk/apidocs/pluggable-widgets-config-api) [**FINISH CROSS-REFERENCE**]).
```
Comment (Adam-Dupaski) [inline content/en/docs/releasenotes/studio-pro/9/9.18.md]: @ConnorLand do you have the cross-reference link that can go here?

Comment (ConnorLand) [reply content/en/docs/releasenotes/studio-pro/9/9.18.md]: I've added the crosslink, which refers to a new section in this 9.18 PR: https://github.com/mendix/docs/pull/5062

File: content/en/docs/releasenotes/studio-pro/9/9.18.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+We see the biggest value here for Design mode, as it gives this mode more control. However, it can be used in combination with Structure mode as well.
+
+With a default Studio Pro layout, the new pane can be found on a tab next to the **App Explorer**. Alternatively, it can be activated in the **View** menu.
+
+#### Widget Developer Console
+
+We added a new feature called the Widget Developer Console for advanced widget development, which helps you debug the configuration code of your widget that is responsible for conditionally visible properties, custom consistency checks, and Structure mode preview. This console shows logs from the `getProperties`, `getPreview`, `check`, and `getCustomCaption` functions, which can be defined inside the new configuration module (for more information, see the [Widget Developer Console](/apidocs-mxsdk/apidocs/pluggable-widgets-config-api)[/apidocs-mxsdk/apidocs/pluggable-widgets-config-api/#widget-dev-console] section of *Configuration Module API for Pluggable Widgets*).
```
Comment (Adam-Dupaski) [inline content/en/docs/releasenotes/studio-pro/9/9.18.md]: @ConnorLand formatting on this cross-reference link looks incorrect?

Comment (ConnorLand) [reply content/en/docs/releasenotes/studio-pro/9/9.18.md]: Sorry about that. It has been fixed.

**Pull request merged**